docs/topics/api/overview.rst

```diff
@@ -1,474 +1,475 @@
-* 2022-08-25: added ``approval_notes`` for version create or edit, and exposed via the version detail response for authorized developers of the add-on and reviewers. https://github.com/mozilla/addons-server/issues/19554
-* 2023-01-05: added ``applications/<application>/`` API endpoint to list all valid appversions for a given application. https://github.com/mozilla/addons-server/issues/20066
-* 2023-01-12: added the ability to delete add-on versions. https://github.com/mozilla/addons-server/issues/19784
-* 2023-02-23: removed the /reviewers/canned-responses/ endpoint. https://github.com/mozilla/addons-server/issues/20354
-* 2023-02-23: removed ``canned-response`` from the /draft_comments/ endpoint. https://github.com/mozilla/addons-server/issues/20353
-* 2023-03-02: added specific HTTP 409 status code for add-on/version submissions that already exist
-* 2023-03-02: added support for calling the version detail endpoint using a version number instead of an ``id``.
-* 2023-03-09: added ``is_disabled`` to version detail and update endpoints, for authenticated developers and reviewers. https://github.com/mozilla/addons-server/issues/20142
-* 2023-03-08: restricted ``lang`` parameter to only alphanumeric, ``_``, ``-``. https://github.com/mozilla/addons-server/issues/20452
-* 2023-03-09: added ``host_permissions`` to the response of the version detail endpoint. https://github.com/mozilla/addons-server/issues/20418
-* 2023-04-13: removed signing api from api/v5+ in favor of addon submission api. https://github.com/mozilla/addons-server/issues/20560
-* 2023-06-01: renamed add-ons search endpoint sort by ratings parameter to ``sort=ratings``, ``sort=rating`` is still supported for backwards-compatibility. https://github.com/mozilla/addons-server/issues/20763
-* 2023-06-06: added the /addons/browser-mappings/ endpoint. https://github.com/mozilla/addons-server/issues/20798
-* 2023-06-22: added ``versions`` to blocklist block endpoint. https://github.com/mozilla/addons-server/issues/20748
-* 2023-07-06: added ``is_all_versions`` to blocklist block endpoint. https://github.com/mozilla/addons-server/issues/20857
-* 2023-10-12: added ``reporter_name`` and ``reporter_email`` as two optional alternatives to an authenticated reporter in the abuse api. https://github.com/mozilla/addons-server/issues/21268
-* 2023-10-26: added ``location`` to abuse api. https://github.com/mozilla/addons-server/issues/21330
-* 2023-11-02: removed ``application`` from categories endpoint, flattened ``categories`` in addon detail/search endpoint. https://github.com/mozilla/addons-server/issues/5989
-
-.. _`#11380`: https://github.com/mozilla/addons-server/issues/11380/
-.. _`#11379`: https://github.com/mozilla/addons-server/issues/11379/
-.. _`#11378`: https://github.com/mozilla/addons-server/issues/11378/
-.. _`#11374`: https://github.com/mozilla/addons-server/issues/11374/
+.. _api-overview:

```

Comment:
Looks like you've added Windows-style line ending (\r\n) instead of Unix ones (\n) in that doc ?

Reply:
Gah! git set up problems on the loaner I'm using